app/config/Global.scala

```diff
@@ -36,17 +36,21 @@ import java.time.Clock
 
 import com.google.inject.AbstractModule
 import com.google.inject.name.Names
-import http.VoaHttpClient
+import com.google.inject.name.Names.named
+import com.typesafe.config.ConfigException
+import http.{VoaHttpClient, VoaHttpClientAuth}
 import infrastructure.{RegularSchedule, Schedule, VOABackendWSHttp}
 import org.joda.time.Duration
 import play.api.Mode.Mode
 import play.api.{Configuration, Environment}
 import uk.gov.hmrc.play.config.ServicesConfig
 
 class GuiceModule(environment: Environment,
-                  configuration: Configuration) extends AbstractModule {
+                  configuration: Configuration) extends AbstractModule with ServicesConfig {
   override def configure() = {
 
+    bind(classOf[VoaHttpClientAuth]).annotatedWith(Names.named("VoaNewAuthedBackendHttp")).to(classOf[VoaHttpClientAuth])
```

Comment:
when the type and the definition are the same you do not require a binding here.

Reply:
removed